todo_app/templates/index.html

```diff
@@ -12,7 +12,17 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+          <li> {{item.title}} - {{item.status}} </li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+  <div class="row">
+    <form action="/addItem" method="POST">
+        <h2><label for="name">Enter your TODO item: </label></h2>
```

Comment:
The label's `for` attribute should match the input's `id` attribute.

When done correctly it binds them together so that e.g. if you click on the label, you will end up focused on the input.